cmd/virt-handler/Dockerfile

```diff
@@ -23,9 +23,20 @@ FROM centos:7
 # no-lower than what was used when the binary was built.
 RUN yum -y install wget && \
     wget -P /etc/yum.repos.d https://copr.fedorainfracloud.org/coprs/jmliger/virt7-upstream/repo/epel-7/jmliger-virt7-upstream-epel-7.repo  \
-    && yum -y install libvirt-client \
+    && yum -y install libvirt-client cyrus-sasl-md5 \
```

Comment:
Because, if we run virt-handler as non-root, there is no guarantee, that uids/gids match between the libvirt and the virt-handler container. Therefore we either would have to synchronize the uids/gids, to not expose the socket read-and writeable for anyone. That however is still insecure, since any other container can use the same uids/gids. Therefore, we expose the socket now read/write-able for anyone, but protect it by a password, which is delivered via a kubernetes secret. This way we don't have to synchronize gids/uids and have a more secure solution.